diff --git a/dist/lib/Cli.js b/dist/lib/Cli.js
index 195146107cecec4951a42d59a0a25c7077f753d7..ce194a725a63adb2a36e46f35c8b53a71765508a 100644
--- a/dist/lib/Cli.js
+++ b/dist/lib/Cli.js
@@ -36,7 +36,7 @@ const merge_1 = require("./cmds/merge");
 const fingerprint_1 = require("./cmds/fingerprint");
 // import {play, PlayArgs} from './cmds/play';
 const core_1 = require("@bubblewrap/core");
-// import { play, PlayArgs, playPublish } from './cmds/play';
+const { play, PlayArgs, playPublish } = require('./cmds/play');
 class Cli {
     async run(args) {
         console.log(constants_1.BUBBLEWRAP_LOGO);
@@ -91,14 +91,14 @@ class Cli {
                 return await (0, merge_1.merge)(parsedArgs);
             case 'fingerprint':
                 return await (0, fingerprint_1.fingerprint)(parsedArgs);
-            // case 'play':
-            //   return await play(parsedArgs as unknown as PlayArgs);
-            // case 'playPublish':
-            //   return await play(parsedArgs as unknown as PlayArgs, 'publish');
-            // case 'playVersionCheck':
-            //     return await play(parsedArgs as unknown as PlayArgs, 'versionCheck');
-            // case 'playRetain':
-            //     return await play(parsedArgs as unknown as PlayArgs, 'retain');
+            case 'play':
+              return await play(parsedArgs);
+            case 'playPublish':
+              return await play(parsedArgs, 'publish');
+            case 'playVersionCheck':
+                return await play(parsedArgs, 'versionCheck');
+            case 'playRetain':
+                return await play(parsedArgs, 'retain');
             default:
                 throw new Error(`"${command}" is not a valid command! Use 'bubblewrap help' for a list of commands`);
         }
