diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index ac91e5f598c67477c08a223dfa549d4576708c80..0000000000000000000000000000000000000000
diff --git a/build/src/gaxios.js b/build/src/gaxios.js
index 26d45b5cc6abcb62c16ed468eb4d8c21384d40ce..0ca47ea991350cd09c9ecd61735fef7c5ec111a7 100644
--- a/build/src/gaxios.js
+++ b/build/src/gaxios.js
@@ -25,12 +25,12 @@ const url_1 = require("url");
 const common_1 = require("./common");
 const retry_1 = require("./retry");
 /* eslint-disable @typescript-eslint/no-explicit-any */
-const fetch = hasFetch() ? window.fetch : node_fetch_1.default;
+const fetch = hasFetch() ? (hasWindow() ? window.fetch : globalThis.fetch) : node_fetch_1.default;
 function hasWindow() {
     return typeof window !== 'undefined' && !!window;
 }
 function hasFetch() {
-    return hasWindow() && !!window.fetch;
+    return (hasWindow() && !!window.fetch) || (typeof globalThis !== 'undefined' && typeof globalThis.fetch === 'function');
 }
 function hasBuffer() {
     return typeof Buffer !== 'undefined';
